components/your-history/ActivityCard.js

```diff
@@ -0,0 +1,144 @@
+
+import CardContent from '@material-ui/core/CardContent';
+import Typography from '@material-ui/core/Typography';
+import { ActivityPropTypes } from './PropTypes';
+
+function getFormattedDateCompleted(timestamp) {
+  const date = timestamp.toDate();
+  return format(date, 'MMMM do');
+}
+function getFormattedDateEntered(timestamp) {
+  const date = timestamp.toDate();
+  return format(date, 'P');
+}
+
+const useStyles = makeStyles(theme => ({
+  card: {
+    minWidth: 275,
+    padding: theme.spacing(2),
+  },
+  divider: {
+    marginTop: theme.spacing(2),
+    marginBottom: theme.spacing(2),
+  },
+  group: {
+    marginTop: theme.spacing(2),
+    marginBottom: theme.spacing(2),
+  },
+  label: {
+    fontWeight: theme.typography.fontWeightMedium,
+  },
+  chip: {
+    marginRight: theme.spacing(1),
+  },
+  header: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'flex-start',
+  },
+  description: {
+    marginTop: theme.spacing(4),
+    marginBottom: theme.spacing(4),
+  },
+}));
+
+function ActivityCard(props) {
+  const classes = useStyles();
+
+  const {
+    timestamp,
+    activityTypeLabel,
+    description,
+    briefDescription,
+    dateCompleted,
+    timeSpentInMinutes,
+    difficultyLevel,
+    activityFeeling,
+    whyIfeelThisWay,
+  } = props;
+  return (
+    <Card className={classes.card}>
+      <CardContent>
+        <div className={classes.header}>
+          <Typography variant="body1" noWrap style={{ maxWidth: '50%', fontWeight: 500 }}>
+            {activityTypeLabel}
+          </Typography>
+          <Typography variant="caption" component="p" noWrap>
+            {getFormattedDateEntered(timestamp)}
+          </Typography>
+        </div>
+
+        <Typography variant="h6" component="h2" className={classes.group}>
+          {briefDescription}
+        </Typography>
+        <Typography variant="body2" component="p" className={classes.description}>
+          {description}
+        </Typography>
+        <Grid container className={classes.group}>
+          <Grid item xs={4}>
+            <Typography variant="body2" className={classes.label}>
+              Completed On
+            </Typography>
+            <Typography variant="body1">{getFormattedDateCompleted(dateCompleted)}</Typography>
+          </Grid>
+          <Grid item xs={4}>
+            <Typography variant="body2" className={classes.label}>
+              Difficulty
+            </Typography>
+            <Typography variant="body1">{difficultyLevel}</Typography>
+          </Grid>
+          <Grid item xs={4}>
+            <Typography variant="body2" className={classes.label}>
+              Estimated Time
+            </Typography>
+            <Typography variant="body1" component="span" display="block">
+              {timeSpentInMinutes} Minutes
+            </Typography>
+          </Grid>
+        </Grid>
+        <Divider className={classes.divider} />
+        <div className={classes.group}>
```

Comment:
The feelings stuff isn't required -- would be good to show these conditionally, to avoid:

<img width="1074" alt="Screen Shot 2019-12-07 at 11 46 49 AM" src="https://user-images.githubusercontent.com/44956/70455173-cfecf780-1a79-11ea-9609-e55f5d8d59d2.png">

Reply:
moved it to the activity card for now